pkg/utils/file.go

```diff
@@ -77,6 +77,24 @@ func FileCopy(src string, dst string) error {
 	return os.WriteFile(dst, data, constants.WriteReadReadPerms)
 }
 
+// SetupExecFile copies a file into destination and set it to have exec perms,
+// if destination either does not exists, or is not executable
+func SetupExecFile(src string, dst string) error {
+	if !IsExecutable(dst) {
```

Comment:
Do we need to error/log if the file isn't executable?

Reply:
We don't want to error. This also happens on first install.
We are not going to error on first install, and we prefer to try to 
recover if a previous install failed.
Probably we can add log for a failed previous install, while trying to recover. 
But that log of unusual situations usually go unnoticed into the log file. 
We have logs that go into logs file, usually not seen by the user on stdout.
And we have ux output for the user on stdout (that also go to logs).